### PR TITLE
feat(readme): set up initial readme

### PR DESCRIPTION
**Pull request by hotcheetoqueen**
this is just the start. we'll need to add the link once live, and i'd love hear input on how we should flesh out!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,26 @@
...
+## Dev Team
+Kevin Loughead
+Tanya Komar
+Jessica Perelman
+
+## Built With
+[Create React App](https://github.com/facebook/create-react-app)
+
+## Features
+• Sticky nav that links to sections on the page
```
Comment (kvnloughead) [inline README.md]: These bullet points aren't rendering the items on separate lines. Try using `-`es.

image

File: README.md [hunk 2]
```diff
@@ -1,70 +1,26 @@
...
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
-
-### `yarn build` fails to minify
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+# AntRobtics Landing Page Overhaul
+
+## Dev Team
+Kevin Loughead
```
Comment (kvnloughead) [inline README.md]: The names are all rendering on a single line. To have them on separate lines (and without bullet points) you can add two spaces to the end of each name: `'some name  `. You can use `<br></br>` tags too, but they produce too much of a gap. 

image
image

Comment (kvnloughead) [reply README.md]: This is another option

image

File: README.md [hunk 3]
```diff
@@ -1,70 +1,26 @@
...
+Tanya Komar
+Jessica Perelman
+
+## Built With
+[Create React App](https://github.com/facebook/create-react-app)
+
+## Features
+• Sticky nav that links to sections on the page
+• Animated header
+• Carousel of robot options and holistic gallery
```
Comment (kvnloughead) [inline README.md]: What does 'holistic' mean in this context?

File: README.md [hunk 4]
```diff
@@ -1,70 +1,26 @@
...
+• Cards detailing benefits and listing the team
+• Contact form for customer inquiries
+• Google Map API for location
```
Comment (kvnloughead) [inline README.md]: I think it isn't accurate to say we are using Google Map API. We are using [react-leaflet](https://react-leaflet.js.org/) which uses [OpenStreetMap](https://www.openstreetmap.org/#map=4/38.01/-95.84). I'd suggest including the links.

File: README.md [hunk 5]
```diff
@@ -1,70 +1,26 @@
...
+• Responsive based on multiple breakpoints
+See our [Trello Board](https://trello.com/b/CPA5Lprq/antrobotics) for the full breakdown!
+
```
Comment (kvnloughead) [inline README.md:25]: We'll need a description of the configurability too. I'll write something up for that. Maybe we could add this bullet point to this readme:

- Designed with configurability in mind. See the [configurability readme](config-readme.md) for details.

And then have the config details in a separate file? I think that link will work, but I'm not certain.

**Review by kvnloughead — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (Wondros-User) [reply README.md:25]: Oo great idea, just added!

File: README.md (hunk 4 above)
Comment (Wondros-User) [reply README.md]: Thanks for the heads up—updated!

File: README.md (hunk 2 above)
Comment (Wondros-User) [reply README.md]: updated with two spaces for now

**Comment by Wondros-User** [comment]
@kvnloughead are we good to merge and we can add site link in a new pr once we have it up on gh pages?

File: README.md (hunk 5 above)
Comment (hotcheetoqueen) [reply README.md:25]: Oo great idea, just added!

File: README.md (hunk 4 above)
Comment (hotcheetoqueen) [reply README.md]: Thanks for the heads up—updated!

File: README.md (hunk 3 above)
Comment (hotcheetoqueen) [reply README.md]: changed to 'company' for now, does that seem ok? it seem like it's going to be a lot more picture types than just specific products like the robot options one

File: README.md (hunk 2 above)
Comment (hotcheetoqueen) [reply README.md]: updated with two spaces for now

File: README.md (hunk 1 above)
Comment (hotcheetoqueen) [reply README.md]: fixed!

**Pull request merged**